build/config/android/config.gni

```diff
@@ -17,7 +17,7 @@ if (is_android) {
   if (!defined(default_android_sdk_root)) {
     default_android_sdk_root = "//third_party/android_tools/sdk"
     default_android_sdk_version = "29"
-    default_android_sdk_build_tools_version = "28.0.3"
+    default_android_sdk_build_tools_version = "29.0.1"
```

Comment:
Not 29.0.2?

Reply:
I think the latest version of the SDK build tools is 29.0.1 - https://androidsdkmanager.azurewebsites.net/Buildtools

Reply:
The [platform-tools](https://developer.android.com/studio/releases/platform-tools) are at v29.0.2

Reply:
Doh! You're right.  Thanks!